### ISSUE TITLE
about evaluate.py

### ISSUE THREAD

**Issue by delamoerxy** [hellochick/ICNet-tensorflow#15]
Hello,
Thank you for your great job .I meet a question when I use the evaluate code.Please give me a hand.
At first,I use your train code trained my own dataset with size 512X512X3 and save a model,all things seems to be right so far.But when I use the evaluate code with the saved model,I found that the mIOU result is about 0.004, all the images are predit to label[0].(I set n_classes =5 and label[0] is blackground in my code)

**Comment by hellochick** [comment]
Hey @delamoerxy, If your background takes a large part in image, it is possible that the network cannot learn how to predict other class easily. Maybe you can try ignore your background label ? 
Can you give me some examples of your own dataset? 

Btw, remember to update mean/var and train beta/gamma variable.

**Comment by delamoerxy** [comment]
@hellochick,thank you.I really forgot to update update mean/var and train beta/gamma variable.
My training dataset is 20K,validation dataset is 3K,after 100K iteration the validation mIOU is about 0.6,that's a not bad result for me.
I'm trying to improve the result now.
Thank you very much!

**Comment by delamoerxy** [comment]
@hellochick I'm sorry to bother you again.
I have try many ways to improve the accuracy on my own dataset,but mIOU only from 0.6 improved to 0.64,and the model often  confused with 2 different class in my dataset ( eg. wolf and husky). Could you give me some suggestions on how to fine tune the model.
Thank you for taking your time to answer my questions.

**Comment by hellochick** [comment]
@delamoerxy , Congratulations! Maybe the model is reach the limit, since wolf and husky is really similar. You can try to change the model by adding the number of filters ( e.g. from 128 -> 256 ). I am really curious about your dataset and inference results, can you show some examples ? Thanks al
